lib/functions/settings/RenderSettings.lua

```diff
@@ -0,0 +1,55 @@
+local assign = import("../../assign")
+
+local RenderSettings = {}
+
+setmetatable(RenderSettings, {
+	__tostring = function()
+		return "RenderSettings"
+	end,
+})
+
+local prototype = {}
+
+local metatable = {}
+metatable.type = RenderSettings
+
+function metatable:__index(key)
+	local internal = getmetatable(self).internal
+
+	if internal[key] ~= nil then
+		return internal[key]
+	end
+
+	if prototype[key] ~= nil then
+		return prototype[key]
+	end
+
+	error(string.format("%s is not a valid member of RenderSettings", tostring(key)), 2)
+end
+
+function metatable:__newindex(key, value)
+	local internal = getmetatable(self).internal
+
+	if internal[key] then
```

Comment:
This check should be `internal[key] ~= nil` in case one of the values ends up being `false`